### PR TITLE
Add opm resources and fix 4.7 build

### PR DESCRIPTION
**Pull request by gnufied**
This PR adds resources for using operator-registry to build index images using new format.

It also fixes build failures for 4.7

### CODE REVIEW

File: manifests/4.6/local-storage-operator.v4.6.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
-  name: local-storage-operator.v4.6.0
```
Comment (huffmanca) [inline manifests/4.6/local-storage-operator.v4.6.0.clusterserviceversion.yaml:4]: Why are we changing the 4.6 version when we also have the `local-storage-operator.v4.7.0.clusterserviceversion.yaml` ?

**Review by huffmanca — COMMENTED** [review]
Just one question about the 4.6 manifest.

File: manifests/4.6/local-storage-operator.v4.6.0.clusterserviceversion.yaml (hunk 1 above)
Comment (gnufied) [reply manifests/4.6/local-storage-operator.v4.6.0.clusterserviceversion.yaml:4]: err typo

Comment (gnufied) [reply manifests/4.6/local-storage-operator.v4.6.0.clusterserviceversion.yaml]: Fixed

**Pull request merged**